pages/story/[slug].js

```diff
@@ -1,50 +1,45 @@
 import { useRouter } from "next/router";
 import Image from "next/image";
+import { story_array } from "./story_data";
 
 export default function Story() {
     const router = useRouter();
     const { slug } = router.query;
+    const story = story_array.find((story) => story.href.endsWith(slug));
```

Comment:
nit: rename story_array to stories for better readability

Reply:
cool